gprofiler/profilers/java.py

```diff
@@ -571,11 +584,6 @@ def _init_java_safemode(self, java_safemode: str) -> None:
                 f" --java-safemode={JavaSafemodeOptions.JAVA_EXTENDED_VERSION_CHECKS}"
             )
 
-        if java_safemode == JAVA_SAFEMODE_ALL:
```

Comment:
Why remove this?

`test_java_safemode_parameters()` fails now

Reply:
See in the [commit message](https://github.com/Granulate/gprofiler/pull/265/commits/a756d9f539d82ddab0edc251bf95539dee1176ea) - we do not enforce this anymore so I removed the check & removed the test as well